components/Slider/README.md

```diff
@@ -159,26 +161,49 @@ func didChangeSliderValue(senderSlider:MDCSlider) {
+</ul>
+
+#### UISlider APIs with different names in MDCSlider
 
-- set color for thumb via @c thumbColor
-- set color of track via @c trackColor
+<ul class="icon-list">
+  <li class="icon-list-item icon-list-item">  The UISlider API `minimumTrackTintColor` has an equivalent API `setTrackFillColor:forState:` in </li>
+    MDCSlider.  This API must first be enabled by setting `statefulAPIEnabled = YES`. 
+  <li class="icon-list-item icon-list-item">  The UISlider API `maximumTrackTintColor` has an equivalent API `setTrackBackgroundColor:forState:`</li>
+    in MDCSlider.  This API must first be enabled by setting `statefulAPIEnabled = YES`.
+  <li class="icon-list-item icon-list-item">  The UISlider API `thumbTintColor` has an equivalent API `setThumbColor:forState:` in MDCSlider.  This</li>
+    API must first be enabled by setting `statefulAPIEnabled = YES`.     
 
-New features:
+#### MDCSlider enhancements not in MDCSlider
 
-- making the slider a snap to discrete values via property numberOfDiscreteValues
+<ul class="icon-list">
+  <li class="icon-list-item icon-list-item--link">  MDCSlider can behave as a <a href="https://material.io/components/sliders/#discrete-slider">Material Discrete Slider</a></li> by
+    setting `discrete = YES` and `numberOfDiscreteValues` to a value greater than or equal to 2. Discrete 
+    Sliders only allow their calculated discrete values to be selected as the Slider's value.  If 
+    `numberOfDiscreteValues` is less than 2, then the Slider will behave as a 
+    [Material Continuous Slider](https://material.io/components/sliders/#continuous-slider).
+  <li class="icon-list-item icon-list-item">  For Discrete Sliders, the track tick color is configured with the `setFilledTrackTickColor:forState:` and</li>
+    `setBackgroundTrackTickColor:forState:` APIs.  The filled track ticks are those overlapping the 
+    filled/active part of the Slider.  The background track ticks are found in any other portions of the track.  These 
+    APIs must first be enabled by setting `statefulAPIEnabled = YES`.
+  <li class="icon-list-item icon-list-item">  Track tick marks can be made shown always, never, or only when dragging via the `trackTickVisibility` </li>
+    API.  If `numberOfDiscreteValues` is less than 2, then tick marks will never be shown.
+  <li class="icon-list-item icon-list-item">  An anchor point can be set via `filledTrackAnchorValue` to control the starting position of the filled track.</li>
+  <li class="icon-list-item icon-list-item">  The track can be made taller (or shorter) by setting the value of `trackHeight`. </li>
+</ul>
 
 #### `-accessibilityActivate`
 
-Our implementation closely resembles what UISlider does but it's not an exact match. On an
-`accessibilityActivate` we move one sixth of the amount between the current value and the midpoint value.
+MDCSlider closely resembles what UISlider does but it's not an exact match. On an
```

Comment:
Wordsmithing this a bit, maybe "MDCSlider's behavior is very similar to that of UISlider, but it is not exactly the same"

Reply:
Done.